install/kubernetes/helm/subcharts/prometheus/templates/serviceaccount.yaml

```diff
@@ -7,7 +7,7 @@ imagePullSecrets:
 {{- end }}
 {{- end }}
 metadata:
-  name: prometheus
+  name: prometheus-service-account
```

Comment:
is there upgrade impact with this change?  Can this change be broken out separately so we can revert it if needed when we go through upgrade testing prior to 1.1 release?

Reply:
I removed that commit from this PR and created a new PR with just that change: https://github.com/istio/istio/pull/10362

Reply:
thank you.